### ISSUE TITLE
[Accessibility Bug] Work unit or project team(s)? items not focusable

### ISSUE THREAD

**Issue by AminaSalah** [CDH-Studio/UpSkill#1549]
**Describe the bug**
Contact Information Form:
For “Work unit or project team(s)?” there doesn’t seem to be an accessible way for the user to tab to the options that are added to the input box. E.g., the user types in “First” and presses enter. Then the user types in “Second” and presses enter. They are unable to tab to the desired option they want to delete if needed. They can only press backspace and delete the last item in the list of items in the input box.

**To Reproduce**

1. Go to Contact Information Form
2. Insert some items into the “Work unit or project team(s)?” input
3. Attempt to tab to a specific item that was inserted

**Expected behavior**
It would be helpful if a user could tab to the exact item that they want to remove.
![cant tab](https://user-images.githubusercontent.com/86394925/125220523-e6ca8000-e294-11eb-8486-ec3028782f0f.jpg)

**Comment by dynamic11** [comment]
The new react-select component replacement should resolve this:
https://react-select.com/creatable
